engagement_and_scoping.qmd

```diff
@@ -7,34 +7,54 @@ The draft currently has no official status. It is a work in progress and is subj
 # Engagement and scoping
 
 ## Introduction
-If the main purpose of analysis is to cut out irrelevance and to simplify, it is essential for to identify what is relevant for a piece of analysis. The first stage of the analytical cycle is initial customer engagement and scoping identifying what the Commissioner requires and so what is relevant for the analysis.  The Analyst works with the  Commissioner to develop sufficient understanding of the problem to design a requisite analysis. Problem Structuring Methods, such as the Strategic Choice Approach, Rich Pictures and Systems Thinking can aid these discussions.
+The first stage of the analytical cycle is initial customer engagement and scoping identifying what the Commissioner requires and so what is relevant for the analysis.  The Analyst works with the  Commissioner to develop sufficient understanding of the problem to design a requisite analysis.
 
-During engagement and scoping , the Commissioner and the Analyst shape the analysis by developing a shared understanding of the problem and the context. They should work together to design analysis that suits the Commissioner’s requirements.
+During engagement and scoping, the Commissioner and the Analyst shape the analysis by developing a shared understanding of the problem and the context. This shared understanding will be used as the basis for designing analysis that suits the Commissioner’s requirements.
 
-## The Commissioner's responsibilities during engagement and scoping
+### The Commissioner's responsibilities during engagement and scoping
 
 The Commissioner is responsible for ensuring that:
 
 * Key aspects of the problem, scope, and programme constraints, are captured and clearly communicated to the Analyst.    
 * They are available to actively engage with the Analyst to appropriately shape the work.    
 * There is sufficient governance in place to support the analysis and its role in the wider project or programme. This is particularly important if the analysis supports business critical decisions. This proportionality may need to be revisited at the design stage if a novel or riskier approach is required (for example if AI models are used).    
 * They understand risks where time and resource pressures constrain the approach.    
-* They request information on uncertainty from analysts and challenge them when it is absent, inadequate or ambiguous.    
-* Communicate to the analyst any sources of uncertainty they have identified as part of their wider considerations.    
-* If possible, indicate in advance the consequences for decision-making of different degrees of uncertainty, as this may enable the analyst to conduct their analysis at a proportionate level.    
-* Commissioner should sign off on the specification
+* They request information from the Analyst on accounting for uncertainty, and challenge them when it is absent, inadequate or ambiguous.    
```

Comment:
In my experience the commissioners seldom have much of an idea about sources of uncertainty or acceptable levels of uncertainty.

Might need to make this explicit.